plenum/config.py

```diff
@@ -149,7 +149,7 @@
 
 
 # Log configuration
-logRotationWhen = 'W'
+logRotationWhen = 'W6'
```

Comment:
Add test to check this value as it is done in node now

Reply:
@dsurnin done